docs/api.md

```diff
@@ -196,6 +196,92 @@ Modifies permissions for the package based on a single or an array of `JSON`-enc
 | member     | The user is a member of the package but cannot administer it |
 | admin      | The user is an admin of the package                          |
 
+### Docker package auto-creation
+
+Adding each Docker package separately can be tedious. AutoDocker package auto-creation offers a way to specify a pattern by which, on the first pull of the image, the package is created automatically.
```

Comment:
```suggestion
Adding each Docker package separately can be tedious. Docker package auto-creation offers a way to specify a pattern by which, on the first pull of the image, the package is created automatically.
```

Reply:
Let's also maybe call out that we will redirect on any pattern match, to indicate that the whole pattern is "available"